source/MapDetailPanel.cpp

```diff
@@ -122,13 +122,31 @@ void MapDetailPanel::Draw()
 
 
 
+bool MapDetailPanel::Scroll(double dx, double dy)
+{
+	Point point = UI::GetMouse();
+	if(excessPlanet && point.X() < Screen::Left() + 160 && point.Y() > Screen::Top() + 90 && point.Y() < Screen::Bottom() - 230)
+	{
+		if(dy > 0. && firstPlanet < excessPlanet)
+			++firstPlanet;
+		else if(dy < 0. && firstPlanet > 0)
+			--firstPlanet;
+		return true;
```

Comment:
This snaps to the next item without any actual _scroll_, which is not exactly stellar UX.

Reply:
I like it to be one system at a time, scrolling is a big word indeed but having half of planets would mean complication for finding where you clicked that would be completely unecessary
most importantly, having half of planets view would simply be unbearable in terms of how it looks according to me

but I'm open to changes, I thought about having it scroll really, with the trade prices on the right and the same scrolling UI as for outfitterPanel etc, with planets being shown with their images and same stuff as now. However that would be too much information and in most of the times, useless, because only systems with more than 7 planets would use it and it would be weird looking to have that scrolling background sometimes be there or not, the idea of the system view is for it to be minimalist according to me.

Reply:
also for a true scroll one more variable, offset, would be required